### PR TITLE
Rpm/vendor v4.14.0

### PR DESCRIPTION
**Pull request by sharvit**
Updating vendor to the latest version, required for: https://github.com/theforeman/foreman/pull/7896

### CODE REVIEW

File: packages/foreman/nodejs-theforeman-builder/nodejs-theforeman-builder.spec [hunk 1]
```diff
@@ -26,157 +26,159 @@ Source12: https://registry.npmjs.org/@babel/helper-explode-assignable-expression
...
+Source150: https://registry.npmjs.org/to-fast-properties/-/to-fast-properties-2.0.0.tgz
+Source151: https://registry.npmjs.org/unicode-canonical-property-names-ecmascript/-/unicode-canonical-property-names-ecmascript-1.0.4.tgz
+Source152: https://registry.npmjs.org/unicode-match-property-ecmascript/-/unicode-match-property-ecmascript-1.0.4.tgz
+Source153: https://registry.npmjs.org/unicode-match-property-value-ecmascript/-/unicode-match-property-value-ecmascript-1.2.0.tgz
+Source154: https://registry.npmjs.org/unicode-property-aliases-ecmascript/-/unicode-property-aliases-ecmascript-1.1.0.tgz
+Source155: nodejs-theforeman-builder-%{version}-registry.npmjs.org.tgz
+%if 0%{?scl:1}
 BuildRequires: %{?scl_prefix_nodejs}npm
-%if 0%{!?scl:1}
+%else
```
Comment (evgeni) [inline packages/foreman/nodejs-theforeman-builder/nodejs-theforeman-builder.spec:171]: that's the part that's breaking it on el8, please revert.

**Comment by sharvit** [comment]
Thanks @evgeni, updated.

I am using npm2rpm v6.0.0 downloaded from npm.
Should I use a different version so I won't need to revert every time I run the script?

**Comment by evgeni** [comment]
There is no release with https://github.com/theforeman/npm2rpm/commit/bfcbb9d939a256e426b78a65b88101f2bd5940bf yet.

@ekohl could you do a release of mpm2rpm please?

**Review by evgeni — APPROVED** [review]

**Comment by ekohl** [comment]
> @ekohl could you do a release of npm2rpm please?

Forgot to update here, but https://github.com/theforeman/npm2rpm/releases/tag/6.0.1

**Pull request merged**